monitor/monitor.cpp

```diff
@@ -246,12 +247,27 @@ int dnet_monitor_process_cmd(struct dnet_net_state *orig, struct dnet_cmd *cmd,
 	DNET_LOG_DEBUG(n, "monitor: {}: {}: process MONITOR_STAT, categories: {:x}, monitor: {:p}",
 	               dnet_state_dump_addr(orig), dnet_dump_id(&cmd->id), req->categories, (void *)n->monitor);
 
+	if (req->backends_number * sizeof(uint32_t) != cmd->size - sizeof(struct dnet_monitor_stat_request)) {
```

Comment:
It is better to make it:`cmd->size != sizeof(struct dnet_monitor_stat_request) + req->backends_number * sizeof(uint32_t)`, so it compares size of packet with sum of its contents.